### PR TITLE
New db approach x new data

### PR DESCRIPTION
**Pull request by richfitz**
This PR is a messy merge of the new db approach and the new data, merging into the epic branch.

Branch now very poorly named as we also change the processing location as we were running the gha runners out of memory

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,17 @@ See the [spec](inst/schema/spec.md) for more details.
 
 ## Updating data
 
+The basic flow of data coming in:
+
+1. New raw (ish) data will come from the science team, recently at `\\fi--didenas1.dide.ic.ac.uk\malaria\Arran\malaria_projects\MINT\1_ModelSimulations\output\malariasimulation_runs\remove_peak_irs_increase_population`. It's really very slow to work with these against the network drive so it's good to copy these locally on your machine for processing.
+1. Edit `inst/version` to create a new data version - should be an ISO date (`YYYYMMDD`)
```
Comment (EmmaLRussell) [inline README.md:31]:
```suggestion
1. Edit `inst/version` to create a new data version - should be an ISO date (`YYYYMMDD`). Typically this date should match the date of the raw data files. 
```

File: README.md [hunk 2]
```diff
@@ -25,6 +25,17 @@ See the [spec](inst/schema/spec.md) for more details.
...
+1. Edit the top of `scripts/import` to reflect the new data
+1. Run `./scripts/import path/to/data path/to/output` to produce a file `<date>.tar`
+1. Copy that file to the network share (recent versions are about 500MB)
+1. RDP to `fi--didex1` and copy the tar file to `C:\xampp\htdocs\mrcdata\mint\<date>.tar`
+1. Update the date field in `R/import.R`
```
Comment (EmmaLRussell) [inline README.md:36]: I don't think it needs to have this, as `import.R` is using inst/version now?

**Review by EmmaLRussell — COMMENTED** [review]
OK, so the actual processing of the data, which used to take place in `mintr_test_db_init` on CI, now has to be done manually on a dev machine, and the output copied to the network drive as described in the readme file change?

The image works fine when run against the app. 

I haven't attempted to verify the network copy location etc. I don't think I've ever had much luck getting on the network share. Is there an idiots guide?

File: README.md [hunk 3]
```diff
@@ -25,6 +25,17 @@ See the [spec](inst/schema/spec.md) for more details.
 
 ## Updating data
 
+The basic flow of data coming in:
```
[the remark this answers is not in the document]
Comment (richfitz) [reply README.md:28]: Not used any more, I'll delete that and tidy up a it

**Comment by richfitz** [comment]
https://mrc-ide.github.io/didehpc/articles/didehpc.html#mapping-network-drives - these instructions tend to work ok, but you also need a login didex1 - this should get much nicer once we have things in orderly/outpack/packit

**Review by EmmaLRussell — APPROVED** [review]

**Pull request merged**